tracer/test/test-applications/security/Samples.Security.AspNetCore2/Samples.Security.AspNetCore2.csproj

```diff
@@ -27,6 +27,7 @@
     <Compile Include="..\Samples.Security.AspNetCore5\Data\**\*.cs" Link="Data\%(RecursiveDir)%(Filename)%(Extension)" />
     <Compile Include="..\Samples.Security.AspNetCore5\Models\**\*.*" Link="Models\%(RecursiveDir)%(Filename)%(Extension)" />
     <Compile Include="..\Samples.Security.AspNetCore5\IdentityStores\**\*.*" Link="IdentityStores\%(RecursiveDir)%(Filename)%(Extension)" />
+    <Compile Include="..\..\Samples.Shared\TraceAttribute.cs" Link="TraceAttribute.cs" />
```

Comment:
I think you want to remove this and the new TraceAttribute.cs file right?

Reply:
Removed. Using the existing one in Annotations